src/Utilities/Testing.hpp

```diff
@@ -220,6 +220,35 @@ namespace GridKit
       return fail == 0;
     }
 
+    /**
+     * @brief Equatlity comparison between maps with a tolerance for the scalar value
+     *
+     * @tparam IdxT
+     * @tparam RealT
+     *
+     * @param[in] std::map<IdxT, RealT> - first map
```

Comment:
Is this doxygen formatting correct? I think this should be the name of the parameter - not its type, right?

Reply:
I also think it's not necessary to annotate `param[in]` for non-pointer arguments but also I guess it doesn't hurt.

Reply:
type --> variable name fixed in [3ea65fb](https://github.com/ORNL/GridKit/pull/89/commits/3ea65fb2a569789533cf3a667568af94f6b69563).